### PR TITLE
test(system-e2e): Disable/fix broken tests

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
Some tests are breaking 100%. This PR `skip`s some (for now), and fixes others.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts [hunk 1]
```diff
@@ -9,24 +9,34 @@ const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
...
 
 test.describe('MS - Skírteini', () => {
-  let context: BrowserContext
+  let contextFæreyjar: BrowserContext
+  let contextAmeríka: BrowserContext
+  const contexts: BrowserContext[] = []
 
   test.beforeAll(async ({ browser }) => {
-    context = await session({
+    contextAmeríka = await session({
```
Comment (brynjarorng) [inline apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts:17]: Is it normal to use non-ascii characters in code that is not related to user readable output?

Comment (AndesKrrrrrrrrrrr) [inline apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts:17]:
```suggestion
    context🌎 = await session({
```

File: apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts [hunk 2]
```diff
@@ -9,24 +9,35 @@ const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
...
+    contextAmerika = await session({
       browser: browser,
       storageState: 'service-portal-amerika.json',
       homeUrl,
       phoneNumber: '0102989',
       idsLoginOn: true,
     })
+    contextFaereyjar = await session({
+      browser: browser,
+      storageState: 'service-portal-færeyjar.json',
```
Comment (brynjarorng) [inline apps/system-e2e/src/tests/islandis/service-portal/smoke/licenses.spec.ts:26]:
```suggestion
      storageState: 'service-portal-faereyjar.json',
```
Also this here?

**Review by brynjarorng — CHANGES_REQUESTED** [review]

**Review by brynjarorng — APPROVED** [review]
LGTM
